modules/sildisco/public/metadata.php

```diff
@@ -144,7 +143,7 @@
 
     if ($idpmeta->hasValue('OrganizationName')) {
         $metaArray['OrganizationName'] = $idpmeta->getLocalizedString('OrganizationName');
-        $metaArray['OrganizationDisplayName'] = $idpmeta->getLocalizedString(
+        $metaArray['OrganizationDisplayName'] = $idpmeta->getOptionalLocalizedString(
             'OrganizationDisplayName',
             $metaArray['OrganizationName']
```

Comment:
Just to leave a note here: this was actually a bug since the SSP 2.0 upgrade changed the definition of `getLocalizedString` by removing the 2nd parameter. I missed this one during the upgrade effort earlier this year. Editorial: things like this make me scared to use interpreted languages.